clang-tools-extra/docs/clang-tidy/checks/daedalean-class-methods.rst

```diff
@@ -0,0 +1,10 @@
+.. title:: clang-tidy - daedalean-class-methods
+
+daedalean-class-methods
+=======================
+
+Functions MUST be implemented outside class definition
+
+Exceptions
+
+Member functions of class templates
```

Comment:
Hm, and what about templatized member functions of class non-templates?

Reply:
Implement outside. Only reason for such exception is tricky way to implement template members outside in some cases (at least template struct declared inside template class, template specializations. Could be more cases that I don't remember)